app/src/main/java/com/pvp/app/ui/screen/steps/StepViewModel.kt

```diff
@@ -40,22 +39,47 @@ class StepViewModel @Inject constructor(
 
     @RequiresApi(Build.VERSION_CODES.UPSIDE_DOWN_CAKE)
     fun updateTodaysSteps() {
+        viewModelScope.launch {
+            val end = LocalDate
+                .now()
+                .plusDays(1)
+                .atStartOfDay(ZoneId.systemDefault())
+                .toInstant()
+            val start = LocalDate
```

Comment:
Variables that are being manipulated in any way (initialized, assigned) with multiple lines block of code should be separated by a single line to increase readability.
```kotlin
            val end = LocalDate
                .now()
                .plusDays(1)
                .atStartOfDay(ZoneId.systemDefault())
                .toInstant()

            val start = LocalDate
                .now()
                .atStartOfDay(ZoneId.systemDefault())
                .toInstant()
```